eslint.config.mjs

```diff
@@ -69,12 +65,6 @@ export default tsEslint.config({
     "sonarjs/no-element-overwrite": "error",
     "sonarjs/no-identical-conditions": "error",
     "sonarjs/no-identical-expressions": "error",
-    "sonarjs/new-cap": "off",
-    "sonarjs/different-types-comparison": "off",
-    "sonarjs/sonar-prefer-regexp-exec": "off",
-    "sonarjs/function-return-type": "off",
-    "sonarjs/no-misleading-array-reverse": "off",
-    "sonarjs/slow-regex": "off",
```

Comment:
There's no reason you should be changing the rules. These should be restored. 

Even if these were from development pulled in they never should have made it in. Can you confirm the reason why these are removed?